pkg/rancher-desktop/main/commandServer/__tests__/settingsValidator.spec.ts

```diff
@@ -17,6 +17,7 @@ const cfg = _.merge(
     kubernetes:  { version: '1.23.4' },
     application: { pathManagementStrategy: PathManagementStrategy.Manual },
   });
+// cfg.virtualMachine.memoryInGB ||= getDefaultMemory();
```

Comment:
Yes. The problem is that it was initially set to `0` in `defaultSettings`, but that caused problems (like the one here), so I init it to 2, which is what we did at the beginning. The implementation replaces it by calling `getDefaultMemory()` in `createSettings`, but otherwise `2` is a good-enough value.